SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -654,9 +656,31 @@ void USpatialNetDriver::Shutdown()
 		}
 	}
 
+	const bool bDeleteDynamicEntities = GetDefault<ULevelEditorPlaySettings>()->GetDeleteDynamicEntities();
```

Comment:
have you considered wrapping this in #if WITH_EDITOR?

Reply:
if it's intended for runtime as well, will that setting be available?

Reply:
It isn't available - I'll update